setup_go.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -ex
+
+# Install gimme to get go1.15.11
+curl -sL -o /bin/gimme https://raw.githubusercontent.com/travis-ci/gimme/master/gimme
+chmod +x /bin/gimme
+eval $(gimme 1.15.11)
+
+# Install gcc 4.9 (needed to compile go, as it introduces -fdiagnostics-color which is required
+# in the post-compilation steps of go)
+# We don't add it to the PATH so that it's not used in the Agent / rtloader builds
+apt-get update && apt-get install -y gcc-4.9-backport
+
+git clone https://go.googlesource.com/go goroot && cd goroot
```

Comment:
Should we delete `goroot` (except for `/goroot/bin`) once the build is finished? (Or, alternatively, do this on a separate build stage and just copy the binary)

Reply:
I can try doing this. However, `gimme` does install the equivalent of everything that's in `goroot` when you use it to install a go version (and not only the binaries), so I wonder if it has a specific reason to do so.

Reply:
It looks like the official Go tar.gz for Linux includes the source code and some more stuff (e.g. IANA timezone database info), so it's probably better to keep it. Marking this as solved.